tests/test_read_simple_rust.py

```diff
@@ -43,3 +43,32 @@ def test_read_simple_file(text='''println!("Hello world");
 ::std::mem::drop''')
     ]))
     compare(jupytext.writes(nb, 'rs'), text)
+
+
+def test_read_write_script_with_metadata_241(rsnb="""#!/usr/bin/env scriptisto
+// ---
+// jupyter:
+//   jupytext:
+//     text_representation:
+//       extension: .rs
+//       format_name: light
+//       format_version: '1.4'
+//       jupytext_version: 1.1.2
+//   kernelspec:
+//     display_name: Rust
+//     language: rust
+//     name: rust
+// ---
+
+let mut a: i32 = 2;
+a += 1;
+"""):
+    nb = jupytext.reads(rsnb, 'rs')
+    assert 'executable' in nb.metadata['jupytext']
+    rsnb2 = jupytext.writes(nb, 'rs')
+
+    # remove version information
+    def remove_version_info(text):
+        return '\n'.join([line for line in text.splitlines() if 'version' not in line])
```

Comment:
For this, typically I use the fixture `no_jupytext_version_number` (you just need to add that argument to the test function, and remove the version information in the text)... If that's not clear I can also do it from here.

Reply:
Thanks, I didn't know this feature exists. I push a new version to use it.

FYI: I copied [this test](https://github.com/mwouts/jupytext/blob/bcf1a7b1fd5bc5f73d6d38460dcd0ce93a4f179e/tests/test_read_simple_python.py#L537), which is where I found this `remove_version_info` function.

Reply:
Thanks! Oh interesting, later on I will update that other test.